pkg/config/load.go

```diff
@@ -112,3 +112,19 @@ func OperateOnCIOperatorConfigDir(configDir string, callback func(*cioperatorapi
 		return nil
 	})
 }
+
+type CompoundCiopConfig map[string]*cioperatorapi.ReleaseBuildConfiguration
+
+func (compound *CompoundCiopConfig) add(handledConfig *cioperatorapi.ReleaseBuildConfiguration, handledElements *FilePathElements) error {
+	(*compound)[handledElements.Filename] = handledConfig
```

Comment:
I've never seen that one before... why do we need to deref?

Reply:
I was getting a `type *CompoundCiopConfig does not support indexing` error message. After some reading, it seems that one does not really need to use pointers to maps (if I understand it correctly, map variables are basically pointers by default), so I'll update the receiver.

Reply:
Or there is no need to have a method at all. Just `make()` the map and pass it.